ForwardingDecision.cs

```diff
@@ -35,14 +35,7 @@ public abstract class ForwardingDecision {
     /// </summary>
     public sealed class Drop : ForwardingDecision {
       // We only need to keep a single instance of Drop, so we use a singleton pattern.
-      public static readonly Drop inst;
-      private Drop () {}
```

Comment:
That's odd, the code went through the compiler, and the Tallyer (which 
IIRC drops packets) ran fine. I'm not sure how the code could have gone 
through the compiler in the first place, because without a constructor 
it can't be possible to call "new" on the class. Does it compile and run 
at your end?

On 2016-07-12 18:22, TMVector wrote:
> In ForwardingDecision.cs [1]:
> 
>> @@ -35,14 +35,7 @@ public abstract class ForwardingDecision {
>> /// </summary>
>> public sealed class Drop : ForwardingDecision {
>> // We only need to keep a single instance of Drop, so we use a
> singleton pattern.
>> - public static readonly Drop inst;
>> - private Drop () {}
> 
> I'm guessing it wasn't your intention to remove the private
> constructor?
> 
> --
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub [2], or mute the
> thread [3].
> 
> 	*
> 
> Links:
> ------
> [1] https://github.com/niksu/pax/pull/7#discussion_r70481435
> [2]
> https://github.com/niksu/pax/pull/7/files/5061c2776a722543741b9a8435a0e2d990fdc225..1da9c43dce25f0ad034cb50045b9e5733305b467#r70481435
> [3]
> https://github.com/notifications/unsubscribe/ABDTTv3l8fBWUkNHGj6wRAN7NWOW5md8ks5qU806gaJpZM4JKIVi

-- 
http://www.cl.cam.ac.uk/~ns441/